engine/test-src/org/pentaho/di/repository/bad-repositories.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@lucboudreau actually looks like the build isn't set up for a separate test-res folder